connector/connect/server/src/main/scala/org/apache/spark/sql/connect/config/Connect.scala

```diff
@@ -213,4 +205,19 @@ object Connect {
     .version("3.5.0")
     .intConf
     .createWithDefault(200)
+
+  val CONNECT_ENRICH_ERROR_ENABLED =
```

Comment:
Is this a static conf? Or configurable per session?

Reply:
Configureable per session may be better since turning off the flag can make the error handling faster (i.e. without an additional rpc)